frb_dart/lib/src/ffi/web.dart

```diff
@@ -116,10 +117,21 @@ abstract class FlutterRustBridgeWireBase {
 typedef WireSyncReturnStruct = List<dynamic>;
 
 extension WireSyncReturnStructExt on WireSyncReturnStruct {
-  Uint8List get buffer => this[0];
+  Uint8List? get buffer => this[0];
   bool get isSuccess => this[1];
 }
 
+int getPlatformUsize(Uint8List data) {
+  return ByteData.view(data.buffer).getUint32(4);
+}
+
+Tuple2<int, int> parseOpaquePtrAndSizeFrom(Uint8List data) {
+  return Tuple2(
+    ByteData.view(data.buffer).getUint32(4),
+    ByteData.view(data.buffer).getUint32(4 + pointerLength),
+  );
+}
```

Comment:
So, are we sure it is always 32bit on web?

If so, maybe extract a constant instead of magic numbers

Reply:
`Unsupported operation: Uint64 accessor not supported by dart2js.`
https://github.com/dart-lang/sdk/issues/10275